media/inf/legcydrv.inf

```diff
@@ -41,7 +41,7 @@ LegacyDriverClassName = "Controladores incompatibles con PnP"
 LegacyDriverClassName = "Sterowniki niezgodne z Plug and Play"
 
 [Strings.0418]
-LegacyDriverClassName = "Modúle pilot de cuplare statică (non-PnP)"
+LegacyDriverClassName = "Drivere ce nu sunt de tipul Conectează și utilizează (non-PnP)"
```

Comment:
Where is that string displayed in ros? Somewhere in the device manager maybe. I do recommend to attach a picture, that the new very long string is not displayed in a cropped manner. All other changes except this one do look ok to me.

Reply:
I will look on it later.

Reply:
I don't know how to reproduce it on my ReactOS VM. Yes, you're right, it's in Device manager. However, I can add a link with a picture from the internet. I will reduce the number of characters.

https://filestore.community.support.microsoft.com/api/images/c1138ca1-9ce7-4ecd-ae5e-2a6f2b4ee893?upload=true

Reply:
I thought more times how could I reproduce it in order to send you a screenshot, but no way, I don't have any special device in order to show me something similar as in the link I sent you. Hence, I can't send you any picture taken by me, but only from the internet. I didn't check the length before. I just took the file and modified it. However, I revised my translation and now there are less than 80 characters.